extensions/positron-python/src/test/positron/session.unit.test.ts

```diff
@@ -114,7 +114,7 @@ suite('Python Runtime Session', () => {
         } as Partial<JupyterAdapterApi>) as JupyterAdapterApi;
 
         sinon.stub(vscode.extensions, 'getExtension').callsFake((extensionId) => {
-            if (extensionId === 'vscode.kallichore-adapter' || extensionId === 'vscode.jupyter-adapter') {
```

Comment:
@jmcphers was this maybe an oversight that this still said `vscode.kallichore-adapter`? I want to highlight that I have changed it here to `positron.positron-adapter`.

Reply:
It's an oversight, but should be `positron.positron-supervisor`.